src/components/Dialog/DialogHelper.tsx

```diff
@@ -2,19 +2,24 @@ import React from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
 import { DialogProps, DialogSize } from './Dialog.types';
 import { Dialog } from './Dialog';
-import { generateId } from '../../shared/utilities';
+import { canUseDocElement, generateId } from '../../shared/utilities';
 
 const uniqueId: string = generateId();
 
 function createWrapperAndAppendToBody(wrapperId: string): HTMLDivElement {
-  const existingElement = document.getElementById(wrapperId) as HTMLDivElement;
-  if (document.getElementById(wrapperId)) {
-    return existingElement;
+  if (canUseDocElement()) {
```

Comment:
can you early exit? Is there anything valuable happening in here before document exists?